presentation/src/main/java/com/example/healthysalad/StartScreen.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.healthysalad
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [StartScreen.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class StartScreen : Fragment() {
+    // TODO: Rename and change types of parameters
```

Comment:
Before creating PR, You should resolve the TODO comments.
If you want to get some advice for implementing them through the PR, you should add some comments for them.